src/components/Header/Header.tsx

```diff
@@ -0,0 +1,105 @@
+import * as React from 'react'
+import styled, { css } from 'styled-components'
+
+import { InjectedProps, withTheme } from '../../hocs/withTheme'
+import { SmartHRLogo } from '../SmartHRLogo/SmartHRLogo'
+import { HeaderButton } from './HeaderButton'
+import { HeaderDropDown, HeaderDropDownMenuProps } from './HeaderDropDown'
+import { HeaderNotification } from './HeaderNotification'
+
+interface Props {
+  tenantName?: string
+  logoUrl?: string
+  helpButtonLink?: string
+  employeeListButtonLink?: string
+  userMailAddress?: string
+  employeeButtonMenu?: HeaderDropDownMenuProps[]
+  userButtonMenu?: HeaderDropDownMenuProps[]
+  notificationNumber?: number
+}
+
+const HeaderComponent: React.FC<Props> = ({ ...props }) => (
+  <Wrapper {...props}>
+    <HeaderLogoArea>
+      <HeaderLogo href={props.logoUrl ? props.logoUrl : '/'} aria-label="SmartHR">
+        <SmartHRLogo />
+      </HeaderLogo>
+      <TenantName {...props}>{props.tenantName}</TenantName>
+    </HeaderLogoArea>
+
+    <HeaderAreaNavi>
+      <HeaderButton url={props.helpButtonLink} icon="fa-question-circle" target="_blank">
+        ヘルプ
+      </HeaderButton>
+
+      <HeaderButton url={props.employeeListButtonLink} icon="fa-th-list">
+        従業員リスト
+      </HeaderButton>
+
+      <HeaderDropDown
+        key="headerDropDown-0"
+        dropDownKey="manage"
+        icon="fa-users"
+        menus={props.employeeButtonMenu}
+      >
+        従業員管理
+      </HeaderDropDown>
```

Comment:
Abstract menu content 🙏

Reply:
If this component is abstracted, the function of Header may not be the same in each product.
So I stopped abstracting this component and hardcoded it.